packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1068,9 +1123,68 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return this._summarizer;
     }
 
-    private get summariesDisabled(): boolean {
-        return this.runtimeOptions.summaryOptions.disableSummaries === true ||
-            this.runtimeOptions.summaryOptions.summaryConfigOverrides?.disableSummaries === true;
+    private readonly summariesDisabled: boolean;
+    private IsSummariesDisabled(): boolean {
```

Comment:
nit: function name should start with lowercase letter. Same comment for IsHeuristicsDisabled.